pkg/controller/controller_test.go

```diff
@@ -409,6 +427,21 @@ func getTestClusterServicePlan() *v1alpha1.ClusterServicePlan {
 	}
 }
 
+func getTestRemovedClusterServicePlan() *v1alpha1.ClusterServicePlan {
```

Comment:
Do we have a test for adding a service / plan, I'd assume that reconcile tests already had those?

Reply:
Yep - we are covered for new services/plans